ITEM.tsv

```diff
@@ -5742,18 +5742,18 @@ ITEM_20150717_005741	It was named as the appearance of a necklace. There are key
 ITEM_20150717_005742	It was named as the appearance of a necklace. There are keys of hidden places among these type necklaces. However, the place is not special, though the necklace is better special.
 ITEM_20150717_005743	There is a transmission that the first Master Sage crafted it.
 ITEM_20150717_005744	Long ago, when one magician's attempt to come back to the world as an Undead failed, only the magic remained in his bones. Other magicians used these bones to make the objects with magical power in them and this is one of their results
-ITEM_20150717_005745	Misfortune is said to follow this necklace but that is not true. Focus on the strenth it give its wearer. 
+ITEM_20150717_005745	Misfortune is said to follow this necklace but that is not true. Focus on the strength it give its wearer. 
```

Comment:
Awkward sentence, would check Korean text.
Maybe "This necklace is said to bring misfortune, but that is untrue. Rather, it brings strength to its wearer."